proxy/mempool/mempool_schedule.py

```diff
@@ -133,10 +201,12 @@ def has_valid_state(self) -> bool:
     def _actual_state(self) -> MPSenderTxPool.State:
         if self.is_empty():
             return self.State.Empty
-        elif self.is_processing():
-            return self.State.Processing
+        elif (self.pending_nonce or 0) < (self.top_tx.nonce or 0):
+            return self.State.Suspended
         elif self._state_tx_cnt != self.top_tx.nonce:
             return self.State.Suspended
+        elif self.is_processing():
+            return self.State.Processing
         return self.State.Queued
 
```

Comment:
@afalaleev Can you please check `_actual_state()`? Since I removed the force-move, the tests break in unusual ways. After debugging, I remembered your comment about suspending on gapped nonce.